### ISSUE TITLE
Inclusive XR workshop

### ISSUE THREAD

**Issue by wseltzer** [w3c/strategy#175]
Possible workshop on inclusive XR, to explore how immersive technologies can support a diverse range of use cases and needs.

**Comment by dontcallmedom** [comment]
Draft CfP now ready for initial review https://w3c.github.io/inclusive-xr-workshop/

**Comment by iherman** [comment]
What bothers me is: where is the Web in this? Looking at the purpose, the list of topics, etc, there is no mention of the Web, and as a cursory reader I could deduce that it is all right to submit any paper that is on XR, even if it does not have anything to do with Web Technologies, Web usage, etc. What is it that W3C wants to bring to the table? (After all, VR/XR etc. is a pretty crowded field...)

**Comment by joshueoconnor** [comment]
@iherman has a good point, maybe we need to fine tune it and work out specifics around the Inclusive XR web? In this space, the more explicit we can be the better as it can be nebulous.

**Comment by dontcallmedom** [comment]
@LJWatson and I have the following reasons to keep the discussion not too tightly coupled with the Web:
* We can (and should) learn from the work that has already been done around VR that is not browser-based and that means inviting people from that world to be part of this workshop.
* we don't want the workshop to be too browser focused. W3C already produces specifications where the primary implementations are not browsers (ePub for example), and this could be the case with XR too.
* we want to explore whether existing W3C specifications are already being used to make XR more accessible on other platforms/devices (like SSML is used by home voice assistants for example).
* Using the term "web" might put people off participating. The term means so many different things, is more or less interchangeable with the "internet" for many people, and even for the cogniscenti it can mean a wide variety of things.

So yes, we would be happy to accept papers on inclusive XR that don't focus specifically on the usage of the technology of the Web - for better or for worse, the community on that topic is not all that large in the first place.

**Comment by samuelweiler** [comment]
Given the analysis of privacy issues the Immersive Web Working Group has already made and the observation that assistive technologies often expose their users to additional fingerprinting, it might be worth having some discussions specific to privacy in the workshop, perhaps as level-setting?    @michael-n-cooper

**Comment by michael-n-cooper** [comment]
Copying @LJWatson as I'm not involved in the planning of the workshop.

**Comment by michael-n-cooper** [comment]
APA review of this workshop complete, no comments.

**Comment by wseltzer** [comment]
https://www.w3.org/2019/08/inclusive-xr-workshop/

**Comment by r12a** [comment]
Looking at the sections on the purpose of the workshop and identifying gaps, inclusive design seems to exclude internationalisation, which is disappointing.

**Comment by wseltzer** [comment]
Report: https://www.w3.org/2019/08/inclusive-xr-workshop/report.html
